app/victoria-metrics/Makefile

```diff
@@ -25,3 +25,6 @@ victoria-metrics-arm:
 
 victoria-metrics-arm64:
```

Comment:
Use pure Go builds for  `-arm` Makefile rules. This will remove dependencies on cross-compilers (`CC=....` env var) and simplify builds